config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
```

Comment:
- [ ] You should use Postgres as db not the default ssqlite3, using Postgres will help you when deploying your app, It is going to be something like this :point_down: 

```ruby
...

default: &default
  adapter: postgresql
  encoding: unicode

...
```